tasks/main.yml

```diff
@@ -11,6 +11,12 @@
     pkg: php-redis
```

Comment:
I believe installing this package is extraneous given that `php{{ php_version }}-redis` is being introduced.